website/docs/tutorials/04-compiling.md

```diff
@@ -31,6 +31,9 @@ bar = [
 ]
 ```
 
+### Contract name
+A contract name refers to an attribute name in this `"protostar.contracts"` section of the configuration file. Some Protostar's features may expect from you providing contract names. In the example above, `foo` and `bar` are contract names.
```

Comment:
This section is a bit complicated to me. Why not simply edit the `The following configuration tells Protostar to create two contracts — 'foo' and 'bar'.` statement instead?

Reply:
I needed to introduce the `Contract name` term somewhere to be able to refer to it. I think this is the right place.

Reply:
It sound weird though (and is incorrect, we polens tend to overuse p.cont.).
I would write it as:

>  Config in the example above defines two contracts. The **contract names** of those contracts are `foo` and `bar`.

Stating definitions formally makes it much harder to read.

Also @mkaput @kasperski95 WDYT about 
`name` -> `alias`

contract alias sounds better to me as a word

Reply:
Agreed, still I would rewrite this section.

Reply:
I don't think it's good idea, better to keep information close to the story and in the boundaries of the story.